rollup/bundle_prelude.js

```diff
@@ -9,8 +9,7 @@ if (!shared) {
 } else if (!worker) {
     worker = chunk;
 } else {
-    var workerBundleString = "var __evalFinished = false; self.onerror = function() { if(!__evalFinished) { console.error('An error occurred while parsing the WebWorker bundle. This is most likely due to improper transpilation by Babel please see https://docs.mapbox.com/mapbox-gl-js/api/#transpiling-v2'); } }; var sharedChunk = {}; (" + shared + ")(sharedChunk); (" + worker + ")(sharedChunk); __evalFinished = true;"
-
+    var workerBundleString = "var __evalFinished = false; self.onerror = function() { if(!__evalFinished) { console.error('An error occurred while parsing the WebWorker bundle. This is most likely due to improper transpilation by Babel; please see https://docs.mapbox.com/mapbox-gl-js/api/#transpiling-v2'); } }; var sharedChunk = {}; (" + shared + ")(sharedChunk); (" + worker + ")(sharedChunk); __evalFinished = true;"
```

Comment:
Since we're overwriting `onerror` anyway, would this work? Also, this won't interfere with error handlers, will it? Like the difference between `window.addEventListener(...)` and then removing the _specific_ callback, instead of overwriting `onerror`?

```suggestion
    var workerBundleString = "self.onerror = function() { console.error('An error occurred while parsing the WebWorker bundle. This is most likely due to improper transpilation by Babel; please see https://docs.mapbox.com/mapbox-gl-js/api/#transpiling-v2'); }; var sharedChunk = {}; (" + shared + ")(sharedChunk); (" + worker + ")(sharedChunk); self.onerror = null;"
```

Which expands to:

```js
self.onerror = function() {
    console.error('An error occurred while parsing the WebWorker bundle. This is most likely due to improper transpilation by Babel; please see https://docs.mapbox.com/mapbox-gl-js/api/#transpiling-v2');
};
var sharedChunk = {};
(" + shared + ")(sharedChunk);
(" + worker + ")(sharedChunk);
self.onerror = null;
```

Reply:
I had to double-check, but `onerror` doesn't overwrite events added with `addEventListener`. Would this technically overwrite someone's global event handler unnecessarily?

Reply:
Since its the worker bundle all the code that runs inside that context is our code, so I don't think there's a risk of overwriting users' error handlers.

Reply:
Ah, then I've sort of just code-golfed it, but hopefully it remains clear enough. 👍